packages/plugins/test/mocha/cli/deploy.spec.ts

```diff
@@ -257,7 +257,7 @@ describe('Deploy CLI command', () => {
       // ensures that the signer can execute a deployment without having a very high balance of ETH.
       await setBalance(
         wallet.address,
-        ethers.toBeHex(ethers.parseEther('0.01')),
+        ethers.toBeHex(ethers.parseEther('1.25')),
```

Comment:
Needed to bump the initial amount of ETH in the signer's wallet because it needs to deploy the System contracts. Does this number sound right to you @RPate97? 1 ETH was too low.

Reply:
Yeah, I think this is reasonable.